packages/boot/test/bootstrapTests/liquidation.ts

```diff
@@ -217,6 +219,39 @@ export const makeLiquidationTestContext = async t => {
   };
 };
 
+export const addSTARsCollateral = async t => {
+  const { controller, buildProposal } = t.context;
+
+  t.log('building proposal');
+  const proposal = await buildProposal({
+    package: 'builders',
+    packageScriptName: 'build:add-STARS-proposal',
```

Comment:
Out of scope, but we should parameterize this function to `addNewCollateral` and take the collateral name. @Chris-Hibbert needed (and made) such a thing in  https://github.com/Agoric/agoric-sdk/pull/8398/files#diff-3c4081d1865d2d24a7abaee93a075a8852065c32783fbe25265746c13441875bR38-R45